tests/test_recipients_viewsets.py

```diff
@@ -0,0 +1,129 @@
+from unittest.mock import Mock
+
+import pytest
+
+from django.core.cache import caches
+from django.urls import reverse
+
+from wagtail_newsletter.campaign_backends.mailchimp import MailchimpCampaignBackend
+from wagtail_newsletter.viewsets import Audience, AudienceSegment
+
+
+class MockMailchimpCampaignBackend(MailchimpCampaignBackend):
+    def __init__(self):
+        self.client = Mock()
+
+
+@pytest.fixture(autouse=True)
+def memory_cache(settings):
+    settings.CACHES = {
+        "default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
+    }
+    caches["default"].clear()
+
+
+@pytest.fixture(autouse=True)
+def backend(monkeypatch: pytest.MonkeyPatch):
```

Comment:
Just out of curiosity: why do we use Pytest's own monkey patching tools here, rather than the ones from the Python stdlib? :slightly_smiling_face: 

(I find the latter probably easier to grasp and better documented. And it would also allow developers who have little experience in Pytest to still understand these tests, if they have experience with classic `unittest` testing)

Reply:
I'm using the pytest monkeypatching fixture because it takes care of removing the monkeypatch when the test is torn down.